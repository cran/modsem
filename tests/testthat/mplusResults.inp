TITLE:
Running Model via Mplus
DATA:
FILE = "mplusResults_2f55eb38256759e6ebc9b0699ecf1d13.dat";
 
VARIABLE:
NAMES = x1 x2 z1 z2 y1 y2; 
 MISSING=.;
 
ANALYSIS:
estimator = ml;
type = random;
algorithm = integration;
process = 8 ;

MODEL:
X BY x1;
X BY x2;
Z BY z1;
Z BY z2;
Y BY y1;
Y BY y2;
Y ON X;
Y ON Z;
Y ON XZ;
XZ | X XWITH Z;

