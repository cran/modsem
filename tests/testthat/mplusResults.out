Mplus VERSION 8.10 DEMO (Linux)
MUTHEN & MUTHEN
07/03/2024  11:10 AM

INPUT INSTRUCTIONS

  TITLE:
  Running Model via Mplus
  DATA:
  FILE = "mplusResults_2f55eb38256759e6ebc9b0699ecf1d13.dat";

  VARIABLE:
  NAMES = x1 x2 z1 z2 y1 y2;
   MISSING=.;

  ANALYSIS:
  estimator = ml;
  type = random;
  algorithm = integration;
  process = 8 ;

  MODEL:
  X BY x1;
  X BY x2;
  Z BY z1;
  Z BY z2;
  Y BY y1;
  Y BY y2;
  Y ON X;
  Y ON Z;
  Y ON XZ;
  XZ | X XWITH Z;




INPUT READING TERMINATED NORMALLY




Running Model via Mplus

SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                        2000

Number of dependent variables                                    6
Number of independent variables                                  0
Number of continuous latent variables                            4

Observed dependent variables

  Continuous
   X1          X2          Z1          Z2          Y1          Y2

Continuous latent variables
   X           Z           Y           XZ


Estimator                                                       ML
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-02
    Relative loglikelihood change                        0.100D-05
    Derivative                                           0.100D-02
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Maximum number of iterations for H1                           2000
Convergence criterion for H1                             0.100D-03
Optimization algorithm                                         EMA
Integration Specifications
  Type                                                    STANDARD
  Number of integration points                                  15
  Dimensions of numerical integration                            2
  Adaptive quadrature                                           ON
Cholesky                                                       OFF

Input data file(s)
  mplusResults_2f55eb38256759e6ebc9b0699ecf1d13.dat
Input data format  FREE


SUMMARY OF DATA

     Number of missing data patterns             1


COVARIANCE COVERAGE OF DATA

Minimum covariance coverage value   0.100


     PROPORTION OF DATA PRESENT FOR Y


           Covariance Coverage
              X1            X2            Z1            Z2            Y1
              ________      ________      ________      ________      ________
 X1             1.000
 X2             1.000         1.000
 Z1             1.000         1.000         1.000
 Z2             1.000         1.000         1.000         1.000
 Y1             1.000         1.000         1.000         1.000         1.000
 Y2             1.000         1.000         1.000         1.000         1.000


           Covariance Coverage
              Y2
              ________
 Y2             1.000



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    1.023       0.003      -2.527    0.05%       0.101      0.778      1.017
            2000.000       1.141      -0.189       4.324    0.05%       1.293      1.954
     X2                    1.215       0.035      -1.619    0.05%       0.472      0.938      1.188
            2000.000       0.797      -0.139       4.004    0.05%       1.445      2.001
     Z1                    1.011      -0.065      -3.220    0.05%       0.067      0.779      1.041
            2000.000       1.184       0.108       4.845    0.05%       1.289      1.896
     Z2                    1.206       0.007      -1.681    0.05%       0.443      0.985      1.203
            2000.000       0.828      -0.063       4.256    0.05%       1.427      1.976
     Y1                    1.180       0.749      -2.850    0.05%      -0.137      0.688      1.046
            2000.000       2.646       1.489       9.583    0.05%       1.418      2.344
     Y2                    1.335       0.657      -3.235    0.05%       0.247      0.937      1.250
            2000.000       1.739       1.317       8.185    0.05%       1.550      2.276


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       22

Loglikelihood

          H0 Value                      -13374.972

Information Criteria

          Akaike (AIC)                   26793.945
          Bayesian (BIC)                 26917.165
          Sample-Size Adjusted BIC       26847.269
            (n* = (n + 2) / 24)



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 X        BY
    X1                 1.000      0.000    999.000    999.000
    X2                 0.814      0.018     44.445      0.000

 Z        BY
    Z1                 1.000      0.000    999.000    999.000
    Z2                 0.814      0.018     45.396      0.000

 Y        BY
    Y1                 1.000      0.000    999.000    999.000
    Y2                 0.806      0.010     82.774      0.000

 Y          ON
    X                  0.678      0.033     20.440      0.000
    Z                  0.578      0.032     18.080      0.000
    XZ                 0.720      0.031     23.387      0.000

 Z        WITH
    X                  0.204      0.025      8.190      0.000

 Intercepts
    X1                 1.023      0.024     42.827      0.000
    X2                 1.215      0.020     60.908      0.000
    Z1                 1.011      0.024     41.563      0.000
    Z2                 1.206      0.020     59.257      0.000
    Y1                 1.034      0.033     30.870      0.000
    Y2                 1.216      0.027     44.762      0.000

 Variances
    X                  0.968      0.040     24.448      0.000
    Z                  1.013      0.041     24.694      0.000

 Residual Variances
    X1                 0.172      0.018      9.540      0.000
    X2                 0.155      0.012     12.417      0.000
    Z1                 0.171      0.018      9.269      0.000
    Z2                 0.157      0.013     12.365      0.000
    Y1                 0.182      0.021      8.563      0.000
    Y2                 0.137      0.014      9.795      0.000
    Y                  0.965      0.041     23.750      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.134E-01
       (ratio of smallest to largest eigenvalue)


     Beginning Time:  11:10:15
        Ending Time:  11:10:18
       Elapsed Time:  00:00:03


Mplus VERSION 8.10 DEMO (Linux) has the following limitations:
  Maximum number of dependent variables: 6
  Maximum number of independent variables: 2
  Maximum number of between variables: 2
  Maximum number of continuous latent variables in time series analysis: 2


MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2023 Muthen & Muthen
